src/Fantomas/CodePrinter.fs

```diff
@@ -4552,10 +4557,45 @@ and genMemberDefn astContext node =
     | MDOpen s -> !-(sprintf "open %s" s)
     // What is the role of so
     | MDImplicitInherit (t, e, _) ->
+        let genBasecall =
+            let shortExpr = genExpr astContext e
+
+            let longExpr =
+                match e with
```

Comment:
You are doing a bit much here, I would go for a more focused pattern match to make your test pass.
There are no tests for the cases where you would have a `LetOrUses` or `Sequential`.
This is fine, but again don't write more code than necessary.

I'd go with something like:
```fsharp
match e with 
| Paren (lpr, (Tuple(es, tr)), rpr, pr) ->
                        sepNln
                        +> indent
                        +> sepOpenTFor lpr
                        +> sepNln
                        +> (col (sepComma +> sepNln) es (genExpr astContext) |> genTriviaFor SynExpr_Tuple tr)
                        +> unindent
                        +> sepNln
                        +> unindent
                        +> sepCloseTFor rpr pr
                        |> genTriviaFor SynExpr_Paren pr
| _ -> genExpr astContext e
```

This is sudo code, I don't know if this snippet will work but you get the idea.
Please also notice that I've add function calls to `genTriviaFor`, this is necessary if the parenthesis or tuple were to have an trivia (like code comments).